src/main/java/ca/bc/gov/ag/efax/mail/service/MailService.java

```diff
@@ -55,29 +55,30 @@
 import com.microsoft.schemas.exchange.services._2006.types.TargetFolderIdType;
 import com.microsoft.schemas.exchange.services._2006.types.holders.ServerVersionInfoHolder;
 
-import ca.bc.gov.ag.exception.MailException;
-import ca.bc.gov.ag.model.MailMessage;
-import ca.bc.gov.ag.model.SentMessage;
-import ca.bc.gov.ag.pdf.PdfService;
-import ca.bc.gov.ag.repository.SentMessageRepository;
-import ca.bc.gov.ag.util.StringUtils;
+import ca.bc.gov.ag.efax.mail.config.ExchangeProperties;
+import ca.bc.gov.ag.efax.mail.model.MailMessage;
+import ca.bc.gov.ag.efax.mail.model.SentMessage;
+import ca.bc.gov.ag.efax.mail.repository.SentMessageRepository;
+import ca.bc.gov.ag.efax.mail.util.StringUtils;
+import ca.bc.gov.ag.efax.pdf.service.PdfService;
+import ca.bc.gov.ag.efax.ws.exception.FAXSendFault;
 import ca.bc.gov.jag.ews.proxy.ExchangeWebServiceClient;
 
 @Service
-public class MailService {
-
+// FIXME: techdept - codeclimate reports this class is too large.
```

Comment:
Small typo in comment
```suggestion
// FIXME: techdebt - codeclimate reports this class is too large.
```

Reply:
:) Fantastic.  I'll make these changes to my next PR.
Thanks!